### PR TITLE
added a blog to be published on 15 Sep WIP

### PR DESCRIPTION
**Pull request by aryangupta701**

### CODE REVIEW

File: site/content/blog/2023-09-15-browser-recorder/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+summary: "ZAP has introduced a new feature to record pre-task activities such as logging in etc. using Browser Recorder."
+type: post
+tags:
+- blog
+- browser recorder
+- client
+- GSoC 2023
+date: "2023-09-15"
+authors:
+    - aryangupta701
```
Comment (psiinon) [inline site/content/blog/2023-09-15-browser-recorder/index.md:12]: Do you need to add your author details in this PR?

Comment (aryangupta701) [reply site/content/blog/2023-09-15-browser-recorder/index.md:12]: no

**Review by psiinon — APPROVED** [review]

**Comment by psiinon** [comment]
WIP which we discuss other options...

**Comment by aryangupta701** [comment]
By the way would be better to add this flag variable for future?

**Comment by thc202** [comment]
Closing, the temporary blog post is no longer needed and there's a PR adding the flag (#2113).

**Pull request closed without merging**